evansmusomi/react-native-102
# Styles Side Drawer Menu

*What this does*
- Styles Side Drawer Menu

*How to verify*
- Navigate to the Find or Share Place screen and open the Side Drawer
- You should see a "Sign Out" menu item with an icon and light grey background
